src/legacy/core_plugins/kibana/index.js

```diff
@@ -152,6 +152,8 @@ export default function (kibana) {
             visualize: {
               show: true,
               createShortUrl: true,
+              edit: true,
```

Comment:
The distinction between `visualize.edit` and `visualize.save` isn't very clear to me. Is there a reason we need both of these? If so, I'd like to rename `save` to something like `create` (and update references accordingly); if not, then I think we could standardize on `save` to be consistent with other apps. What do you think?